pkg/config/config_template.yaml

```diff
@@ -88,7 +88,7 @@ api_key:
 ##  results in the raw tag being transformed into "foo:1", "foo:2", "foo:3" tags
 #
 # tag_value_split_separator:
-#   - <TAG_KEY>: <SEPARATOR>
```

Comment:
So format has changed silently in 7.22 or doc was broken before?

Reply:
The code has always read this as a map. The docs here have always been broken since they were added in https://github.com/DataDog/datadog-agent/pull/3285